CommonMark.Portable45/CommonMark.Portable45.csproj

```diff
@@ -0,0 +1,175 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
the project file should use the exising Base project so that the files are not included in more that one place.